File: RobWork/example/python/ex-load-workcell.py
from sdurw import *
import sys

if __name__ == '__main__':
    if len(sys.argv) != 2:
        print("Usage: python3 " + sys.argv[0] + " <workcell>")
        sys.exit(1)

    workcell = WorkCellLoaderFactory.load(sys.argv[1]);
    if workcell.isNull():
        print("WorkCell could not be loaded.")
        sys.exit(1)

    print("Workcell " + workcell.getName() + " successfully loaded.")
    sys.exit(0)

File: RobWork/example/java/src/ExLoadWorkCell.java
import org.robwork.LoaderRW;
import org.robwork.sdurw.*;

public class ExLoadWorkCell {

    public static void main(String[] args) throws Exception {
        LoaderRW.load("sdurw");

        if (args.length != 1) {
            System.out.print("Usage: java " + ExLoadWorkCell.class.getSimpleName());
            System.out.println(" <workcell>");
            System.exit(1);
        }

        WorkCellPtr workcell = WorkCellLoaderFactory.load(args[0]);
        if (workcell.isNull()) {
            System.out.println("WorkCell could not be loaded.");
            System.exit(1);
        }

        System.out.print("Workcell " + workcell.getName());
        System.out.println(" successfully loaded.");
    }

}
